plugin/storage/es/factory.go

```diff
@@ -218,7 +218,7 @@ func (f *Factory) CreateArchiveSpanReader() (spanstore.Reader, error) {
 	if !f.archiveConfig.Enabled {
 		return nil, nil
 	}
-	sr, err := createSpanReader(f.getArchiveClient, f.archiveConfig, true, f.logger, f.tracer)
+	sr, err := createSpanReader(f.getArchiveClient, f.archiveConfig, f.logger, f.tracer, "archive")
```

Comment:
I think we need to make this new parameter part of the config and pass it that way, otherwise how would v2 be able to do it since we don't want it to call CreateArchiveSpanReader in the future

Reply:
@yurishkuro +1 - would you prefer it done in this PR? I was initially going to create a follow-up for it

Reply:
ok